python_modules/dagster-graphql/dagster_graphql/implementation/fetch_asset_checks.py

```diff
@@ -30,21 +30,26 @@ def _fetch_asset_checks(
     asset_key: AssetKey,
     check_name: Optional[str] = None,
 ) -> GrapheneAssetChecks:
+    asset_graph = ExternalAssetGraph.from_workspace(graphene_info.context)
+
     external_asset_checks = []
     for location in graphene_info.context.code_locations:
-        # check if the code location is too old to support executing asset checks individually
-        code_location_version = (location.get_dagster_library_versions() or {}).get("dagster")
-        if code_location_version and version.parse(code_location_version) < version.parse("1.4.14"):
```

Comment:
since we broke compat < 1.5, this isn't needed. Followup diff can remove the front end.

We could consider doing a check for ≥1.5, and showing an error on the checks tab if not